src/kudu/consensus/log_cache.cc

```diff
@@ -28,7 +28,6 @@
 #include <gflags/gflags.h>
 #include <glog/logging.h>
 #include <google/protobuf/wire_format_lite.h>
-#include <google/protobuf/wire_format_lite_inl.h>
```

Comment:
This was removed in a higher version of protobuf. kuduraft links fine without it so we aren't using anything in it.